### ISSUE TITLE
randomly stops uploading

### ISSUE THREAD

**Issue by haoliu119** [Bostwickenator/STGUploader#12]
When I have hundreds of photos "ready to upload", I hit the "start upload" button and walk away. Then when I come back after sometime I often see the uploader stops uploading after a couple dozen photos, and the "upload status" says complete, when in fact there are still hundreds of photos left "ready to upload". Thanks.

**Comment by tzwalf** [comment]
I have this issue as well. To work around it, I've installed the [OpenMemories App Store](https://sony-pmca.appspot.com/plugin/app/com.github.ma1co.openmemories.appstore), and when I encounter the issue I will remove STGUploader and then reinstall from the App Store - this always resolves the problem. I can't find the exact steps to reproduce, but guaranteed it will happen at least once if I have 100 photos to upload.
Alex, if I can help with testing or running a tweaked version with some logging, just let me know, very happy to contribute in any way I can.
Tristan

**Comment by jonasjeeliasson** [comment]
Hi, 

First off, really awesome app! 👍 

I have sort of this issue as well. Uploaded 120 pics yesterday and all was successfully uploaded. Took 10 new pics today but when I press upload it just shows a spinner for a second or two and then says complete (but still 10 pics ready to upload). Haven't tried to reinstall it thought but sad it failed after just one go.

Any other fixes other then reinstall it? 

Thanks!

**Comment by jonasjeeliasson** [comment]
Found a workaround, removing and reinserting the battery fixed it for me.

**Comment by Bostwickenator** [comment]
@jonasjeeliasson This was due to Google turning off the backend API. Please try again with version 1.7. Hopefully the process will be more reliable on this new backend.

**Comment by Hellmark** [comment]
still having this issue. It is pretty frequent. I'll switch apps, then switch back, and that sometimes helps, but not always.

**Comment by jonasjeeliasson** [comment]
Yeah, same for me. After every upload I just remove and reinsert the battery and it fixes it for next time. 

Feels like it's holding on to an old token or something that gets wiped by removing the battery. So switching between apps might cause it to lose the "broken" token just like reinserting the battery does.

**Comment by ICH88** [comment]
This is still happening unfortunately. Only battery removal or reinstalling the app fixes it.
